os_win/include/cord.h

```diff
@@ -0,0 +1,352 @@
+/*
+ * Copyright (c) 1993-1994 by Xerox Corporation.  All rights reserved.
+ *
+ * THIS MATERIAL IS PROVIDED AS IS, WITH ABSOLUTELY NO WARRANTY EXPRESSED
+ * OR IMPLIED.  ANY USE IS AT YOUR OWN RISK.
+ *
+ * Permission is hereby granted to use or copy this program
+ * for any purpose,  provided the above notices are retained on all copies.
+ * Permission to modify the code and to distribute modified code is granted,
+ * provided the above notices are retained, and a notice that the code was
+ * modified is included with the above copyright notice.
+ */
+
+/*
+ * Cords are immutable character strings.  A number of operations
```

Comment:
folder `os_win` is compiled external dependencies, it has re2 and boehm pre-compiled. There is no easy way to compile them on user machine, I put it here temporary until I manage to write some installation script. I'm looking forward for your changes which are removing these dependencies for scala-native. So, then I don't need to write this installation script.